tests/unit/async_worker/test_enrichment.py

```
@@ -0,0 +1,197 @@
+
+
+class TestConfigDefaults:
+    @pytest.mark.asyncio
+    async def test_uses_config_defaults_when_none(self, base_metadata_item, mock_telegraph):
+        """When store_telegraph/store_document are None, config defaults should be used."""
+        with patch("async_worker.services.enrichment.STORE_TELEGRAPH", True), \
+             patch("async_worker.services.enrichment.STORE_DOCUMENT", False):
+            result = await enrich(base_metadata_item)
+
+        # STORE_TELEGRAPH=True means Telegraph should be called
+        MockTg, instance = mock_telegraph
```

Ruff (0.15.6)

[warning] Unpacked variable `instance` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)